frontend/src/lib/MetricsManagerTestUtils.ts

```diff
@@ -14,17 +14,16 @@
  * limitations under the License.
  */
 
-import { mockSessionInfo } from "./mocks/mocks"
-import { SessionInfo } from "./SessionInfo"
-
 // This file is only used in tests, so these imports can be in devDependencies
 /* eslint-disable import/no-extraneous-dependencies */
-import { SegmentMetricsManager } from "./SegmentMetricsManager"
+
+import { MockMetricsManager, mockSessionInfo } from "./mocks/mocks"
+import { SessionInfo } from "./SessionInfo"
 
 export function getMetricsManagerForTest(
   sessionInfo?: SessionInfo
-): SegmentMetricsManager {
-  const mm = new SegmentMetricsManager(sessionInfo || mockSessionInfo())
+): MockMetricsManager {
+  const mm = new MockMetricsManager(sessionInfo || mockSessionInfo())
```

Comment:
See comments below - I think we ought to be able to just delete this whole file, and replace calls to `getMetricsManagerForTest` with `new MockMetricsManager()`